src/api/index.js

```diff
@@ -36,6 +38,23 @@ const search = searchTerm => new Promise((resolve, reject) => {
   });
 });
 
+const updateConfig = () => new Promise((resolve, reject) => {
```

Comment:
@vutran looks like I had an old fork. I've made a new one to implement the changes